dpgen/generator/arginfo.py

```diff
@@ -101,8 +101,33 @@ def training_args() -> List[Argument]:
 
 
 # Exploration
+def model_devi_jobs_template_args() -> Argument:
+    doc_template = 'Give a template for 01.model_devi.'
+    doc_template_lmp = 'The path to input.lammps template'
+    doc_template_plm = 'The path to input.plumed template'
+
+    args = [
+        Argument("lmp", str, optional=True, doc=doc_template_lmp),
+        Argument("plm", str, optional=True, doc=doc_template_plm),
+    ]
+    return Argument("template", list, args, [], optional=True, repeat=False, doc=doc_template)
+
+
+def model_devi_jobs_rev_mat_args() -> Argument:
+    doc_rev_mat = '' # looking forward to update
+    doc_rev_mat_lmp = '' # looking forward to update
```

Comment:
doc_rev_mat_lmp = 'revise matrix for revising variable(s) defined in the lammps template into the specific values (iteration-resolved).'